generators/server/templates/settings.gradle.ejs

```diff
@@ -61,5 +61,14 @@ gradleEnterprise {
         publishAlways()
     }
 }
+
+buildCache {
```

Comment:
Why not always enable it?

Reply:
We can do so, but from my talk with Nelson from gradle not everyone wants to have it enabled by default, but of course we could do it (I don't see that much of a problem e.g. for our own use case)